src/app/coursegrab/notifications/push_notifications.py

```diff
@@ -27,13 +27,13 @@
 
 # Initialize SendGrid client
 try:
-    sendgrid_client = SendGridAPIClient(environ.get("SENDGRID_API_KEY"))
+    sendgrid_client = SendGridAPIClient(os.environ.get("SENDGRID_API_KEY"))
```

Comment:
very small nit but why do you do os.environ[...] everywhere else and os.environ.get here

Reply:
+1